shimTool/Gui.py

```diff
@@ -774,31 +760,66 @@ def updateAllDisplays(self):
 
     def updateDisplay(self, viewIndex):
         """Update a specific view with the corresponding underlying data available."""
-        viewDataSlice = self.viewDataSlice[viewIndex]  # this should be a 2d numpy array now
-        # if view data is not none, then so should the slice and maxAbs value
+        viewDataSlice = self.viewDataSlice[viewIndex]  # this should be a 2D numpy array now
+
         if viewDataSlice is not None:
             # Extract the slice and normalize it
             scale = self.viewMaxAbs[viewIndex]
```

Comment:
should probably delete all the instances of this viewMaxAbs attribute. i think iiirc this was the only reason that I had made it.